senior-knights/course-schedulizer
# Feature/popover validation

This feature adds form validation to the popover. It requires prefix, number, and section are added. This also makes use of a Form context provider which cuts down on repeated LOC. That's why the additions and deletions are high. 

To test:
- try to click submit on a pop over to see the errors and then try to fix them or pass in bad values. 
- test clicking a created event and then seeing that the data fill properly



Finally, I suggest we make the popover into a full-page in the future because there are so many options that need to be adjusted and we should have a simplified version of editing that is a popover. I'm basing a lot of the design decisions on Google Calendar.
